problems.rb

```diff
@@ -0,0 +1,13 @@
+# TODO: fix this whole function
+def printHiabunchoftimesyay(numberofpotatoess)
```

Comment:
It would be helpful to use clearer and more descriptive/relevant names for both the function and the input.